### ISSUE TITLE
openSUSE makefile wrong path libexec -> lib64

### ISSUE THREAD

**Issue by pseiler** [andwn/marsdev#18]
hi andwn,

I tried to install marsdev but the tools are installed into /home/philipp/mars/m68k-elf/_lib64_/gcc/m68k-elf/10.3.0 instead of  /home/philipp/mars/m68k-elf/_libexec_/gcc/m68k-elf/10.3.0.

I searched a bit for answer in the makefiles etc. but didn't find the solution to fix this.
Many other dependencies have hardcoded libexec so installing to lib64 by default breaks things.

**Comment by andwn** [comment]
I'm not sure what is going on with your computer or what errors you are getting, but the directory should be libexec. If you symlink libexec to lib64 there does it work? Or does it fail to install in the first place?

**Comment by pseiler** [comment]
Installing is just fine. Only some paths are missing when compiling software that depends on libraries in this path.
Symlinking fixed the issue so far.

**Comment by pseiler** [comment]
One thing to mention. Adding _--libdir=$(MARSDEV)/$(TARGET)/lib_ to the configure lines in Makefile located in _toolchain/_ fixes many issues.
But libcc1 still installs into lib64. I cannot find something that forces the path to lib/.
I also commented the removal of the build directories to check for any hints. It seems that "../lib64" is added as $libsuffix by configure scripts. Still not sure why this is created. Something because _/usr/lib64_ and _/lib64_ are the defaults in openSUSE.

I checked my archlinux Installation and  there it works flawlessly.
